website/content/en/docs/_index.md

```diff
@@ -0,0 +1,208 @@
+# The default controller will watch for AppService objects and create a pod for each CR
+$ kubectl create -f deploy/crds/app.example.com_v1alpha1_appservice_cr.yaml
+
+# Verify that a pod is created
+$ kubectl get pod -l app=example-appservice
+NAME                     READY     STATUS    RESTARTS   AGE
+example-appservice-pod   1/1       Running   0          1m
+
+# Test the new Resource Type
+$ kubectl describe appservice example-appservice
+Name:         example-appservice
+Namespace:    myproject
+Labels:       <none>
+Annotations:  <none>
+API Version:  app.example.com/v1alpha1
+Kind:         AppService
+Metadata:
+  Cluster Name:        
+  Creation Timestamp:  2018-12-17T21:18:43Z
+  Generation:          1
+  Resource Version:    248412
+  Self Link:           /apis/app.example.com/v1alpha1/namespaces/myproject/appservices/example-appservice
+  UID:                 554f301f-0241-11e9-b551-080027c7d133
+Spec:
+  Size:  3
+
+# Cleanup
+$ kubectl delete -f deploy/crds/app.example.com_v1alpha1_appservice_cr.yaml
+$ kubectl delete -f deploy/operator.yaml
+$ kubectl delete -f deploy/role.yaml
+$ kubectl delete -f deploy/role_binding.yaml
+$ kubectl delete -f deploy/service_account.yaml
+$ kubectl delete -f deploy/crds/app.example.com_appservices_crd.yaml
+```
+
+**Note:** Follow the steps in the [Getting Started Repository][getting_started] to learn how to develop your Operator projects.
+ 
+## Command Line Interface
+
+To learn more about the SDK CLI, see the [SDK CLI Reference][sdk_cli_ref], or run `operator-sdk [command] -h`.
+
+## User Guides
+
+To learn more about the writing an operator in Go, see the [user guide][guide].
+
+The SDK also supports developing an operator using Ansible or Helm. See the [Ansible][ansible_user_guide] and [Helm][helm_user_guide] operator user guides.
+
+### Operator capability level
+
+Note that each operator type has a different set of capabilities. When choosing what type to use for your project, it is important to understand the features and limitations of each of the project types and the use cases for your operator.
+
+![operator-capability-level](/images/operator-capability-level.png)
+
+## Samples
+
+To explore any operator samples built using the operator-sdk, see the [operator-sdk-samples][samples].
+
+## FAQ
+
+For common Operator SDK related questions, see the [FAQ][faq].
+
+## Contributing
+
+See [the contributor guide][contrib] for details on submitting patches and the contribution workflow.
+
+See the [proposal docs][proposals_docs] and issues for ongoing or planned work.
+
+## Reporting bugs
+
+See [reporting bugs][bug_guide] for details about reporting any issues.
+
+## License
+
+Operator SDK is under Apache 2.0 license. See the [LICENSE][license_file] file for details.
+
+[install_guide]: {{< ref "install-operator-sdk.md" >}}
+[operator_link]: https://coreos.com/operators/
+[proposals_docs]: /docs/proposals
+[sdk_cli_ref]: {{< ref "sdk-cli-reference.md" >}}
+[guide]: /docs/user/
```

Comment:
This link fails the `marker` tests. Alternative syntax is relative, but AFAICT we cannot make `marker` play nice with Hugo. `marker` checks paths to files, but Hugo converts the markdown to html, and the paths to not resolve to files.

To move forward with Hugo, we will need to remove the `marker` tests. If we want to keep link checking, we should be able to add an html checker, and run against built docs. Anyone have a recommendation for such a tool?

Reply:
Would something like this work: https://github.com/raviqqe/muffet?